cromwellApiClient/src/main/scala/cromwell/api/CromwellClient.scala

```diff
@@ -29,12 +29,21 @@ class CromwellClient(val cromwellUrl: URL, val apiVersion: String)(implicit acto
   def abortEndpoint(workflowId: WorkflowId): String = workflowSpecificEndpoint(workflowId, "abort")
   def statusEndpoint(workflowId: WorkflowId): String = workflowSpecificEndpoint(workflowId, "status")
   def metadataEndpoint(workflowId: WorkflowId): String = workflowSpecificEndpoint(workflowId, "metadata")
+  def outputsEndpoint(workflowId: WorkflowId): String = workflowSpecificEndpoint(workflowId, "outputs")
+  def logsEndpoint(workflowId: WorkflowId): String = workflowSpecificEndpoint(workflowId, "logs")
+  def diffEndpoint(workflowA: WorkflowId, callA: String, indexA: ShardIndex, workflowB: WorkflowId, callB: String, indexB: ShardIndex): String = {
+    def shardParam(aOrB: String, s: ShardIndex) = s.index.map(i => s"&index$aOrB=$i.toString").getOrElse("")
```

Comment:
I see the way this is being called it's "Little Bobby Tables"-proof, and I'm pretty sure nothing malicious could happen from passing junk to this "safe" endpoint anyway, but building URLs through interpolation of an unrestricted `String` still makes me somewhat uneasy.  Could this use a proper type (i.e. case class) or some fancy "refined" magic instead?